en-US/emails/2017/FxnU_oct.lang

```diff
@@ -0,0 +1,150 @@
+## NOTE: A series of 4 emails to new subscribers to the Firefox+You mailing list
+## NOTE: Bug 1407074; Polish only
+
+
+# Email 1
+# Subject Line
+;Here’s all the good you’re doing by choosing Firefox
+Here’s all the good you’re doing by choosing Firefox
+
+
+# Preheader
+;You just said “Yes!” to a healthier internet.
+You just said “Yes!” to a healthier internet.
+
+
+# Headline
+;Good for you. Good for the web.
+Good for you. Good for the web.
+
+
+;Hey, nice work choosing Firefox for your web journeys. Not all heroes wear capes — by browsing with us, you have a direct impact on the causes shaping the future of the internet.
+Hey, nice work choosing Firefox for your web journeys. Not all heroes wear capes — by browsing with us, you have a direct impact on the causes shaping the future of the internet.
+
+
+;How? Because by “us,” we mean not just Firefox, but Mozilla, the technology nonprofit that backs us. Mozilla brings internet education to people all over the world, fights for your online rights and keeps corporate power in check.
+How? Because by “us,” we mean not just Firefox, but Mozilla, the technology nonprofit that backs us. Mozilla brings internet education to people all over the world, fights for your online rights and keeps corporate power in check.
+
+
+;You’re already doing good. And together we can do more. Learn how you can rock the power of your internet citizenship.
+You’re already doing good. And together we can do more. Learn how you can rock the power of your internet citizenship.
+
+
+;Thanks,
+Thanks,
```

Comment:
It looks like these two strings are together, either add a comment or use one string: `Thanks,<br>The Firefox Team`.

Reply:
If this is the solution, I will apply the fix to all 4 emails.